src/MessageBox.Avalonia/BaseWindows/MsBoxCustomWindow.cs

```diff
@@ -1,32 +0,0 @@
-using System.Threading.Tasks;
-using Avalonia.Controls;
-using MsWindow = MessageBox.Avalonia.Views.MsBoxCustomWindow;
-
-namespace MessageBox.Avalonia.BaseWindows
-{
-    public class MsBoxCustomWindow : IMsBoxWindow<string>
```

Comment:
By deleting / changing the public API, we thereby call breaking changes

Reply:
@rstm-sf I don't think what it will cause breaking changes.It's just for internal use, User don't call it directly so I may change it easily

Reply:
i think here we just need to change the access modifier of this API because it doesn't need to be used by users